docs/user-guide/cli-db2plugin.md

```diff
@@ -34,9 +34,11 @@ Use one of the following methods to install the the Zowe CLI Plug-in for IBM Db2
 
 ### Installing from an online registry
 
-If you installed Zowe CLI from **online registry**, complete the following steps:
+Complete the following steps if you installed Zowe CLI from **online registry**:
 
-1. Open a command line window and issue the following command:
+1. If you are installing the plug-in on an Apple computer that contains a MacOS M1 processor, complete the steps in [MacOS M1 processor installation](../user-guide/cli-db2-install-m1.md). If not, continue to Step 2.
```

Comment:
Since we are already qualifying that they are using an Apple computer, can we drop `MacOS` from `MacOS M1 Processor`, and just leave it with `M1 Processor`?

Possibly unrelated note, what happens if/when they release another iteration of the processor (i.e. M2)? Will we need to update the documentation to include that?